pkg/network/go/bininspect/symbols.go

```diff
@@ -21,26 +21,40 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+const (
+	symbolChunkSize = uint64(100)
+)
+
 // getSymbolNameByEntry extracts from the string data section the string in the given position.
 // If the symbol name is shorter or longer than the given min and max (==len(preallocatedBuf)) then we return false.
 // preAllocatedBuf is a pre allocated buffer with a constant length (== max length among all given symbols) to spare
 // redundant allocations. We get it as a parameter and not putting it as a global, to be thread safe among concurrent
 // and parallel calls.
-func getSymbolNameByEntry(sectionReader io.ReaderAt, startPos, minLength int, preAllocatedBuf []byte) (string, bool) {
+func getSymbolNameByEntry(sectionReader io.ReaderAt, startPos, minLength int, preAllocatedBuf []byte) int {
 	_, err := sectionReader.ReadAt(preAllocatedBuf, int64(startPos))
 	if err != nil {
-		return "", false
+		return -1
+	}
+
+	foundNull := false
```

Comment:
Please add to the function comment about the new logic

Reply:
the documentation still represents the new logic